optimism/inverse/test/FiniteDifferenceFixture.py

```diff
@@ -0,0 +1,67 @@
+from optimism.test.MeshFixture import MeshFixture
+from collections import namedtuple
+import numpy as np
```

Comment:
For consistency, can you import numpy as `onp` here? We try to be consistent about calling the jax numpy `np` and the original numpy as `onp`. It makes it easier to understand a file at a glance.